src/utils.py

```diff
@@ -3,7 +3,27 @@
 import glob
 import os
 import json
-from inference import FramePoseResult, PersonPoseResult, PoseKeypoint
+from inference import VideoPoseResult, FramePoseResult, PersonPoseResult, PoseKeypoint
+from keypoint_pairs import COCO_TO_MEDIAPIPE, COCO_TO_OPENPOSE
+
+def convert_keypoints_to_coco_format(video_pose_result: VideoPoseResult, model_name: str) -> list:
+    frame_results = video_pose_result.frames
+    if model_name == "YoloPose":
+         return frame_results # Yolo keypoints are already stored in Coco format
+    
+    model_to_coco_mapping = {"MediaPipePose": COCO_TO_MEDIAPIPE, "OpenPose": COCO_TO_OPENPOSE, "mp_pose": COCO_TO_MEDIAPIPE, "openpose_body25b": COCO_TO_OPENPOSE}
+    if model_name not in model_to_coco_mapping:
+         raise ValueError(f"{model_name} is an invalid pose estimators. Acceptable pose estimators are {list(model_to_coco_mapping.keys())} and YoloPose")
```

Comment:
I would suggest to pass the keypoints mapping into this function rather than the model name. This way, we ensure, that we are not fixed to a certain model name, but rather work with the mappings themselves. For example, I would not know, what to pass for model_name for the ground truth of tragic talkers, but I would surely know which mapping to put. This also makes the code in this function way easier, because you do not need to create another dict, check for the mapping etc, but simply work with the given mapping.